packages/react-mutation-mapper/src/component/oncokb/OncoKbCard.tsx

```diff
@@ -1,131 +1,47 @@
-import SummaryWithRefs from './SummaryWithRefs';
 
 import oncoKbLogoImgSrc from '../../images/oncokb_logo.png';
-import collapsibleStyles from './collapsible.module.scss';
-import levelStyles from './level.module.scss';
 import mainStyles from './main.module.scss';
-import tabsStyles from './tabs.module.scss';
-
-type OncoKbCardPropsBase = {
-    title: string;
-    gene: string;
-    variant: string;
-    oncogenicity: string;
-    mutationEffect: string;
-    mutationEffectCitations: Citations;
-    geneSummary: string;
-    variantSummary: string;
-    tumorTypeSummary: string;
-    biologicalSummary: string;
-    treatments: IndicatorQueryTreatment[];
+import OncoKbCardBody from './OncoKbCardBody';
+import { OncoKbCardTitle } from './OncoKbCardTitle';
+import { OncoKbCardDataType } from './OncoKbHelper';
+
+export type OncoKbCardProps = {
+    type: OncoKbCardDataType;
+    hugoSymbol: string;
+    geneNotExist: boolean;
+    isCancerGene: boolean;
+    usingPublicOncoKbInstance: boolean;
     pmidData: ICache;
+    indicator?: IndicatorQueryResp;
     handleFeedbackOpen?: React.EventHandler<any>;
 };
 
-export type OncoKbCardProps =
-    | (OncoKbCardPropsBase & {
-          geneNotExist: boolean;
-          isCancerGene: boolean;
-          usingPublicOncoKbInstance: boolean;
-      })
-    | (Partial<OncoKbCardPropsBase> & {
-          geneNotExist: boolean;
-          isCancerGene: boolean;
-          usingPublicOncoKbInstance: boolean;
-      });
-
-const ONCOKB_DATA_ACCESS_PAGE_LINK =
-    'https://docs.cbioportal.org/2.4-integration-with-other-webservices/oncokb-data-access';
-
-const OncoKbMedicalDisclaimer = (
-    <p className={mainStyles.disclaimer}>
-        The information above is intended for research purposes only and should
-        not be used as a substitute for professional diagnosis and treatment.
-    </p>
-);
-
-const publicInstanceDisclaimerOverLay = (
-    <div>
-        <p>
-            This instance of cBioPortal does not currently have a license for
-            full OncoKB content and is therefore missing therapeutic
-            implications. To obtain a license, please follow{' '}
-            <a href={ONCOKB_DATA_ACCESS_PAGE_LINK} target={'_blank'}>
-                these instructions
-            </a>
-            .
-        </p>
-        {OncoKbMedicalDisclaimer}
-    </div>
-);
-
 @observer
 export default class OncoKbCard extends React.Component<OncoKbCardProps> {
-    @observable activeTab: 'oncogenicity' | 'mutationEffect' = 'oncogenicity';
-    @observable levelsCollapsed: boolean = true;
-
     @computed
     get oncokbLinkOut() {
         let link: string | undefined = undefined;
-        if (this.props.gene) {
-            link = `https://oncokb.org/gene/${this.props.gene}`;
-            if (this.props.variant) {
-                link = `${link}/${this.props.variant}`;
+        if (this.props.hugoSymbol) {
+            link = `https://oncokb.org/gene/${this.props.hugoSymbol}`;
```

Comment:
should be using proxy?   either way maybe we use a function from urls.ts

Reply:
Unfortunately we cannot use a function from `urls.ts` in any of the packages. We need to decouple it from the main project and make it generic first. We can pass the url as a property to this component though. But probably we still need a default value.

Also this is just a static link-out, not an API endpoint. Not sure if a proxy url is necessary here.